attributes/default.rb

```diff
@@ -10,8 +11,5 @@
   'database' => {
     'driver' => 'sqlite3',
     'datasource' => '/var/lib/drone/drone.sqlite'
-  },
-  'registration' => {
-    'open' => true
```

Comment:
Yeah, registration option now goes into each source control type. The readme was updated to show examples.